evaluation/ml_bench/scripts/summarise_results.py

```diff
@@ -7,10 +7,17 @@ def extract_test_results(res_file_path: str) -> tuple[list[str], list[str]]:
     passed = []
     failed = []
     costs = []
+    instance_ids = set()
+    instances = []
     with open(res_file_path, 'r') as file:
         for line in file:
             data = json.loads(line.strip())
             success = data['metrics']['success']
+            if data['instance_id'] in instance_ids:
+                print(f'WARNING: Duplicate instance_id found: {data["instance_id"]}')
```

Comment:
interesting! Do you actually see this happening? I wonder how 🤔  IIRC a task won't run if it has already finished?

Reply:
No, it happened because of the previous error. After fixing it, I don't have this error anymore.

Reply:
Okay, it doesn't harm to do it anyways